src/Umbraco.Web.Common/ApplicationBuilder/UmbracoPipelineFilter.cs

```diff
@@ -2,43 +2,112 @@
 
 namespace Umbraco.Cms.Web.Common.ApplicationBuilder;
 
-/// <summary>
-///     Used to modify the <see cref="IApplicationBuilder" /> pipeline before and after Umbraco registers it's core
-///     middlewares.
-/// </summary>
-/// <remarks>
-///     Mainly used for package developers.
-/// </remarks>
+/// <inheritdoc />
 public class UmbracoPipelineFilter : IUmbracoPipelineFilter
 {
+    /// <summary>
+    /// Initializes a new instance of the <see cref="UmbracoPipelineFilter" /> class.
+    /// </summary>
+    /// <param name="name">The name.</param>
     public UmbracoPipelineFilter(string name)
-        : this(name, null, null, null)
-    {
-    }
+        : this(name, null, null, null, null, null)
+    { }
```

Comment:
Would you want to mark it as obsolete now as well?

Reply:
There's no need to obsolete this constructor, but it can be safely removed in the next major, because that would only be a source breaking change (since it will then use the constructor with optional parameters instead).